{{cookiecutter.repository_slug}}/.gitignore

```diff
@@ -1 +0,0 @@
-ab-results-*
```

Comment:
This gitignore is useful when using the `awesome_bot` which generates this kind of files. Maybe we can put it back and add a comment to say it's to ignore `awesome_bot` results files.

Reply:
we should definitely keep it, you're right!